cmd/go-sbot/main.go

```diff
@@ -120,27 +123,12 @@ func initFlags() {
 	flag.BoolVar(&flagPrintVersion, "version", false, "print version number and build date")
 
 	flag.Parse()
-
-	if debugLogDir != "" {
-		logDir := filepath.Join(repoDir, debugLogDir)
-		os.MkdirAll(logDir, 0700) // nearly everything is a log here so..
-		logFileName := fmt.Sprintf("%s-%s.log",
-			filepath.Base(os.Args[0]),
-			time.Now().Format("2006-01-02_15-04"))
-		logFile, err := os.Create(filepath.Join(logDir, logFileName))
-		if err != nil {
-			panic(err) // logging not ready yet...
-		}
-		logging.SetupLogging(logFile)
-	} else {
-		//logging.SetupLogging(os.Stderr)
-	}
-	//log = logging.Logger("sbot")
-	log = testutils.NewRelativeTimeLogger(nil)
 }
 
 func runSbot() error {
 	initFlags()
+	//log = logging.Logger("sbot")
+	log = testutils.NewRelativeTimeLogger(nil)
```

Comment:
lil bit uncertain about putting this before the invocation of
```
 		logging.SetupLogging(logFile)
```

at the same time, i also wanted to use the structured logging mechanism that all other runtime messages use when making a pronouncement regarding the config (can't find config at "...bla..." / config found at "...bla...")

Reply:
I want to change away from NewRelativeTimeLogger, for long running code it doesn't make sense it's mostly a development convienence (and used in the tests for that reason) but we shouldn't do this in this PR.

Reply:
alright :) i don't know anything about loggers in golang, but i think i get what yr saying! anyway i'll leave it as-is for this PR and maybe inline yr comment for the future